lizmap/modules/lizmap/classes/lizmapFts.class.php

```diff
@@ -11,9 +11,7 @@ class lizmapFts
 {
     protected $sql;
 
-    protected $project;
-
-    protected function setSql()
+    protected function setSql($project)
```

Comment:
I recommend to rename this method to something like `generateSql`, as it is not really a setter method (i.e, the parameter is not a value to set).

Reply:
Method renamed